website/docs/r/route53_resolver_rule_association.html.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "aws"
+page_title: "AWS: aws_route53_resolver_rule_association"
+sidebar_current: "docs-aws-resource-route53-resolver-rule-association"
+description: |-
+  Provides a Route53 Resolver rule association.
+---
+
+# aws_route53_resolver_rule
+
+Provides a Route53 Resolver rule association.
+
+## Example Usage
+
+```hcl
+resource "aws_route53_resolver_rule_association" "example" {
+  resolver_rule_id = "rslvr-rr-0123456789abcdef0"
```

Comment:
For examples it's generally preferred to use reference values from other examples, in this case I would expect to see:

```hcl
resource "aws_route53_resolver_rule_association" "example" {
  resolver_rule_id = "${aws_route53_resolver_rule.example.id}"
  vpc_id           = "${aws_vpc.example.id}"
}